unicef/iogt
# Add user registration, login and update views

### Source 
https://github.com/unicef/iogt/issues/55

### Description
- Add a wagtail page called `UserRegistrationPage` where user can add extra questions to the registration process
- Add customise `User` model
- Add login view
- Add user profile view
- Add update user view
- Add tests

### How to test
1. Drop SQLite database
2. Do fresh migrations
3. Go to wagtail admin and create with custom questions `UserRegistrationPage` (the only one this page can be created per site) 
4. Check if user registration works
5. Check login `/accounts/login/`
6. Check profile update `/accounts/update/`
7. Run tests `./manage.py test --keepdb --settings=iogt.settings.testing`
